### DISCUSSION TITLE
I hope you can help me. I don't have an account to join Slackhelp request:

### DISCUSSION THREAD

**Discussion by cubxxw** [apache/apisix#8502] [Q&A] [answered]
### Description

I hope you can help me. I don't have an account to join Slack

this application is invalid.

my email address is: [3293172751nss@gmail.com](https://mail.google.com/mail/u/0/?fs=1&tf=cm&to=3293172751nss@gmail.com)

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by Hazel6869** [comment, marked as answer]
@cubxxw Hi, you can see https://apisix.apache.org/docs/general/join/

**Comment by cubxxw** [comment]
@Hazel6869I don't have an email address with the suffix `@apache.org`.

Unable to register⚠️

**Comment by tokers** [comment]
You don't need the apache mail. Just register an account with your own email and then accept the invitation.

**Reply by Hazel6869** [reply]
@cubxxw yes, you just need a common email , do you have any question？

**Reply by cubxxw** [reply]
No, thank you. It worked out ok

/close

**Reply by Hazel6869** [reply]
> No, thank you. It worked out ok
> 
> /close

ok you can close this question.
